packages/team-management/package.json

```diff
@@ -49,7 +49,8 @@
   },
   "peerDependencies": {
     "@opensrp/i18n": "^0.0.1",
-    "antd": "^4.6.3",
+    "@testing-library/react": "^12.1.2",
```

Comment:
This is more of a devDependency, though given the approach we have structured the repo, instead of adding it in the packages' package.json, we add it to the root level package.json devDependencies.